src/main/kotlin/me/zeroeightsix/kami/feature/command/PeekCommand.kt

```diff
@@ -36,14 +36,16 @@ object PeekCommand : Command(), Listenable {
     override fun register(dispatcher: CommandDispatcher<CommandSource>) {
         dispatcher.register(
             LiteralArgumentBuilder.literal<CommandSource>("peek").executes { context: CommandContext<CommandSource>? ->
-                val stack = mc.player.inventory.mainHandStack
-            if (ShulkerBoxCommon.isShulkerBox(stack.item)) {
+                val stack = mc.player?.inventory?.mainHandStack
+            if (ShulkerBoxCommon.isShulkerBox(stack?.item)) {
                 val entityBox =
-                    ShulkerBoxBlockEntity(((stack.item as BlockItem).block as ShulkerBoxBlock).color)
-                entityBox.world = mc.world
+                    ShulkerBoxBlockEntity(((stack?.item as BlockItem).block as ShulkerBoxBlock).color)
+                //IDEA is telling me 'Val cannot be reassigned'?
```

Comment:
val is basically a final variable in Java. You can't reassign it, use var instead